tests/output/pjs/output_test.js

```diff
@@ -249,12 +264,18 @@ describe("Scratchpad Output Exec", function() {
         image(rock, 0, 0);
     });
 
-    failingTest("getSound with no string", function () {
+    failingTest("playSound with no string", function () {
         var s = getSound("");
 
         playSound(s);
     });
 
+    failingTest("stopSound with no string", function () {
```

Comment:
What happens if you stop a sound that was never playing or stop a sound multiple times?